solution/backend/regulations/rss_feed.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.syndication.views import Feed
+from django.urls import reverse
+from datetime import datetime
+from regcore.models import Part
+from resources.models import AbstractResource
+
+
+class PartFeed(Feed):
+    def get_feed(self, obj, request):
+        feedgen = super().get_feed(obj, request)
+        feedgen.content_type = 'application/xml'  # New standard
+        return feedgen
+    title = 'Federal Register documents RSS Feed'
```

Comment:
I woud make put the get_feed function after description and a blank line before that.  Its not failling the linter but it should probably be done that way